provisioning/service/src/Config/IndividualEnrollment.cs

```diff
@@ -141,6 +141,7 @@ public IndividualEnrollment(string registrationId, Attestation attestation)
         /// <param name="createdDateTimeUtc">the <code>DateTime</code> with the date and time that the enrollment was created. This is optional and can be <code>null</code>.</param>
         /// <param name="lastUpdatedDateTimeUtc">the <code>DateTime</code> with the date and time that the enrollment was updated. This is optional and can be <code>null</code>.</param>
         /// <param name="eTag">the <code>string</code> with the eTag that identify the correct instance of the enrollment in the service. It cannot be <code>null</code> or empty.</param>
+        /// <param name="capabilities">the <see cref="DeviceCapabilities"/> that identifies if the device is an Edge-enabled device. This is optional and can be <code>null</code>.</param>
```

Comment:
I think the comment shouldn't mention Edge. That is only one of the possibilities but we may be adding more in the future.

Reply:
Ah: this is internal - doesn't matter that much. Still incorrect.